artifacthub/library/pod-security-policy/host-network-ports/1.1.1/samples/psp-host-network-ports/constraint_block_host_network.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sPSPHostNetworkingPorts
+metadata:
+  name: psp-host-network
+spec:
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
+  parameters:
+    hostNetwork: false
```

Comment:
can you add exemptImages here to make sure `hostNetwork: true` aways fails even with exemptImages?

Reply:
done